disconnect-entitylist.json

```diff
@@ -5337,6 +5338,7 @@
       "google.vu",
       "google.ws",
       "ingress.com",
+      "nest.com",
```

Comment:
This will trigger a big entity list update again.

Reply:
Then can we split this PR into smaller chunks? IMHO it'd be great to process the removals separately.

Reply:
No problem. I just created a new PR that omits the google change: https://github.com/mozilla-services/shavar-prod-lists/pull/85.